compiler/src/dotty/tools/dotc/CompilationUnit.scala

```diff
@@ -27,6 +27,12 @@ class CompilationUnit protected (val source: SourceFile) {
   /** Pickled TASTY binaries, indexed by class. */
   var pickled: Map[ClassSymbol, Array[Byte]] = Map()
 
+  /** The fresh name creator for the current unit.
+   *  FIXME: This is not fine-grained enough to enable reproducible builds,
```

Comment:
@smarter could you open an issue for this.

Reply:
done: https://github.com/lampepfl/dotty/issues/7661